waku/node/rest/health/openapi.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.0.3
```

Comment:
can't we have a single `openapi.yaml` for all waku apis?

Reply:
I had the same idea while working on filter rest api... but it would involve dir / code changes, like I would move relevant types to higher level due to duplications. I would leave it for later restructure work.

Reply:
yep makes sense. out of scope for this pr but would love to have everything in the same file at some point.

Reply:
It should be possible to combine the existing openapi.yaml into a single one (which we could publish as part of release?). But I'd say having them separate makes them more maintainable than a single massive yaml?